AutoGVP/00-file_and_variant_check.pl

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/perl
+
+use strict;
+
+################################################################################
+# 00-file_and_variant_check.pl
+# written by Ammar Naqvi
+#
+# This script checks if required files exist and if the variants match up
+#
+# usage: perl 00-file_and_variant_check.pl <optional input dir>
+################################################################################
+
+my $dir_to_check = "";
+my $vcf_file_check = "";
+my $intervar_file_check = "";
+my $multianno_file_check = "";
+my $autopvs1_file_check = "";
+my $dir_to_test = $ARGV[0];
+
+if($dir_to_test=~/\w+/){
+  $dir_to_check = $dir_to_test;
+}
+else{
+  $dir_to_check = "input";
+}
+
+## check to see if input files all exist
+if (glob($dir_to_check."/*.vcf")) {
+  print "vcf file exists...\n";
+  $vcf_file_check = $dir_to_check."/*.vcf";
+
+}
+else {
+  die("FAIL: vcf file does not exist\n");
+}
+
+if (glob($dir_to_check."/*.intervar")) {
+  print "intervar file exists...\n";
+  $intervar_file_check = $dir_to_check."/*.intervar";
+}
+else {
+  die("FAIL: intervar file does not exist\n");
+}
+
+if (glob($dir_to_check."/*hg38_multianno.txt")) {
+  print "multianno file exists...\n";
+  $multianno_file_check = $dir_to_check."/*hg38_multianno.txt";
+}
+else {
+  die("FAIL: multianno file does not exist\n");
+}
+
+if (glob($dir_to_check."/*autopvs1.txt")) {
+  print "autopvs1 file exists...\n";
+  $autopvs1_file_check = $dir_to_check."/*autopvs1.txt";
+
+}
```

Comment:
```suggestion
if (glob($dir_to_check."/*autopvs1*")) {
  print "autopvs1 file exists...\n";
  $autopvs1_file_check = $dir_to_check."/*autopvs1*";

}
```

reducing search to just "autopvs1", in case files are tsv or some other format